components/automate-chef-io/content/docs/ldap.md

```diff
@@ -583,7 +948,7 @@ attribute, multiple queries will be executed, and their results combined.
 
 #### Other Common Issues
 
-If a user, following a login through LDAP or SAML, sees a
+If a user, following a sign in through LDAP or SAML, sees a
```

Comment:
since this common issue is related to LDAP or SAML should it be repeated on the SAML page?

Reply:
also, i think maybe this section heading should be `502 Bad Gateway Errors` or similar since it seems thats all its describing